src/3d/qgscameracontroller.cpp

```diff
@@ -275,6 +278,12 @@ void QgsCameraController::updateCameraFromPose( bool centerPointChanged )
       mCameraPose.setCenterPoint( QgsVector3D( intersectionPoint ) );
       mCameraPose.updateCamera( mCamera );
     }
+    else
+    {
+      QgsVector3D centerPoint = mCameraPose.centerPoint();
+      centerPoint.set( centerPoint.x(), mTerrainEntity->terrainElevationOffset(), centerPoint.z() );
```

Comment:
mTerrainEntity may be null here...

Reply:
There is an if block that checks if mTerrainEntity is null that contains this else block

Reply:
Ah right... sorry